common/src/main/scala/azure/apns/LiveEventProperties.scala

```diff
@@ -0,0 +1,17 @@
+package azure.apns
+
+import play.api.libs.json.Json
+
+case class LiveEventProperties(
+  title: String,
+  body: String,
+  richviewbody: String,
```

Comment:
**very minor detail:** 
Does richviewbody have to be called like this when converted to json or can we make it camel case ?